docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -70,19 +70,44 @@ KUBE_SERVICE_ADDRESSES="--portal_net=10.254.0.0/16"
 KUBE_API_ARGS=""
 ```
 
-* Edit /etc/kubernetes/controller-manager to appear as such:
+* Addition of nodes.
+
+* Create following node.json file on kubernetes master node:
+
+```json
+{
+  "id": "fed-node",
+  "kind": "Minion",
+  "apiVersion": "v1beta1",
+  "resources": {
+    "capacity": {
+      "cpu": 1,
+      "memory": 1073741824
+    },
+  },
+  "labels": {
+    "name": "fed-node-label",
+  },
+}
 ```
-# The following values are used to configure the kubernetes controller-manager
 
-# defaults from config and apiserver should be adequate
+Now create a node object internally in your kubernetes cluster by running:
 
-# Comma separated list of minions
-KUBELET_ADDRESSES="--machines=fed-minion"
+```
+$ kubectl create -f node.json
+
+$ kubectl get nodes
+NAME                LABELS              STATUS
+fed-node           name=fed-node-label     Unknown
```

Comment:
Actually you are right, that it should happen after api-server is started. But it should happen on fed-master, so should be ok to go after services are started.